web/auth/auth.go

```diff
@@ -541,21 +541,32 @@ func authorizeForm(c echo.Context) error {
 			"Error":  "Error Invalid scope",
 		})
 	}
+  readOnly := permissions[0].Verbs.ReadOnly()
 	params.client.ClientID = params.client.CouchID
+  
+  clientURL, err := url.Parse(params.client.ClientURI)
+  var clientDomain string
+  if err != nil {
+    clientDomain = params.client.ClientURI
+  } else {
+    clientDomain = clientURL.Hostname()
+  }
```

Comment:
We want to show the owner's domain, and the client's domain, but we only have the client's URL. Is it ok to sort of ignore the error, since I don't think it's worth showing an error when this happens (can it even happen ???) ?